packages/babel-plugin-transform-regenerator/test/arrow-inside-generator.js

```diff
@@ -0,0 +1,19 @@
+const assert = require("assert");
+const babel = require("babel-core");
+
+test("Doesn't throw with arrow function inside generator", function () {
```

Comment:
I think our tests are all `describe` and `it`.

I think this tests would be best as an `exec.js` test like https://github.com/babel/babel/blob/7.0/packages/babel-plugin-transform-regenerator/test/fixtures/function-sent/example/exec.js